### ISSUE TITLE
Automate tap check on pull requests

### ISSUE THREAD

**Issue by edgarrmondragon** [edgarrmondragon/tap-confluence#6]
- In a pull request, use `singer-check-tap` from `singer-tools` to validate tap output.
- GitHub actions can be used.
- A comment should be automatically created to reflect the validation output:

  ```
  The output is valid.
  It contained 17 messages for 4 streams.                                                                                                                                         
                                                                                                                                                                                
        4 schema messages                                                                                                                                                         
        8 record messages                                                                                                                                                         
        5 state messages                                                                                                                                                          
                                                                                                                                                                                
  Details by stream:                                                                                                                                                              
  +-----------+---------+---------+                                                                                                                                               
  | stream    | records | schemas |                                                       
  +-----------+---------+---------+                                                                                                                                               
  | blogposts | 0       | 1       |                                                                                                                                               
  | groups    | 7       | 1       |                                                                                                                                               
  | spaces    | 1       | 1       |                                                       
  | themes    | 0       | 1       |                                                                                                                                               
  +-----------+---------+---------+
  ```

**Comment by edgarrmondragon** [comment]
Fixed by #7

**Comment by edgarrmondragon** [comment]
Fixed by #7
